sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -58,7 +58,9 @@
         Mapping[str, Any],
         ServiceBusMessage,
         AmqpAnnotatedMessage,
-        List[Union[Mapping[str, Any], ServiceBusMessage, AmqpAnnotatedMessage]],
```

Comment:
It is not safe to pass through typing like this: see https://github.com/python/mypy/issues/3351#issuecomment-300447832

Reply:
I do like the layout now, its cleaner :) . The comment in the mypy issue is about invariants, was the current set up showing that ?

Reply:
Changed to iterable instead of typing as list